online-data-tools/update_www.py

```
@@ -0,0 +1,247 @@
+    for name in STATIC_ASSETS:
+        src = cfg.static_src / name
+        dst = cfg.www_worktree / name
+        shutil.copyfile(src, dst)
+        written.append(dst)
+    return written
+
+
+def stage_sqljs(
+    cfg: Config,
+    *,
+    fetch: Callable[[str], bytes] = lambda url: urllib.request.urlopen(url, timeout=90).read(),
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)